data/tlg0090/tlg001/__cts__.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg0090.tlg001" xml:lang="grc" groupUrn="urn:cts:greekLit:tlg0090">
+    <ti:title xml:lang="grc">Geographiae informatio</ti:title>    
+    
+    <ti:edition xml:lang="grc" urn="urn:cts:greekLit:tlg0090.tlg001.opp-grc1" workUrn="urn:cts:greekLit:tlg0090.tlg001">
+        <ti:label xml:lang="lat">
+            Geographiae informatio, Geographi graeci minores Volumen Secundum
+        </ti:label>
+        <ti:description xml:lang="eng">
+            Agathemerus, creator; Müller, Karl, editor.
+        </ti:description>
+    </ti:edition>
+    
+    <ti:edition urn="urn:cts:greekLit:tlg0090.tlg001.opp-grc3" workUrn="urn:cts:greekLit:tlg0090.tlg001">
+        <ti:label xml:lang="lat">
+            Geographiae informatio, Geographiæ veteris scriptores græci minores Volumen Secundum
+        </ti:label>
+        <ti:description xml:lang="eng">
+            Agathemerus, creator; Hudson, John (1662-1719), editor. 
+        </ti:description>
+    </ti:edition>
+    
+    <ti:edition urn="urn:cts:greekLit:tlg0090.tlg001.opp-grc6" workUrn="urn:cts:greekLit:tlg0090.tlg001">
+        <ti:label xml:lang="eng">
+            Geographiae informatio, Greek Roman and Byzantine Studies
+        </ti:label>
+        <ti:description xml:lang="eng">
+            Agathemerus, creator; Diller, Aubrey (1903-1985), editor and translator.
+        </ti:description>
+    </ti:edition>
+    
+    <ti:edition urn="urn:cts:greekLit:tlg0090.tlg001.opp-grc7" workUrn="urn:cts:greekLit:tlg0090.tlg001">
+        <ti:label xml:lang="lat">
+            Agathemeri hypotyposes geographiae
+        </ti:label>
+        <ti:description xml:lang="eng">
+            Agathemerus, creator; Hoffmann, Samuel Friedrich Wilhelm (1803-1872), editor.
+        </ti:description>
+    </ti:edition>
+    
+    <ti:edition urn="urn:cts:greekLit:tlg0090.tlg001.opp-grc9" workUrn="urn:cts:greekLit:tlg0090.tlg001">
+        <ti:label xml:lang="lat">
+            Agathemeris Orthonis filii compendiariae geographiae expositionum libri duo
+        </ti:label>
+        <ti:description xml:lang="eng">
+            Agathemerus, creator; Tennulius, Samuel (Samuel ten Nuyl), editor and translator.
+        </ti:description>
+    </ti:edition>
+    
+    
+    <ti:edition urn="urn:cts:greekLit:tlg0090.tlg001.opp-grc10" workUrn="urn:cts:greekLit:tlg0090.tlg001">
+        <ti:label xml:lang="lat">
+            Geographica antiqua. Agathemeri Hypotyposis geographiae omnia Graeco-Latina. 
+        </ti:label>
+        <ti:description xml:lang="eng">
+            Agathemerus, creator; Gronovius, Jacobus (1645-1716), editor and translator. 
+        </ti:description>
+    </ti:edition>
+    
+    
+    
+    <ti:translation urn="urn:cts:greekLit:tlg0090.tlg001.opp-eng1" workUrn="urn:cts:greekLit:tlg0090.tlg001">
```

Comment:
I think the problem is that you are required to have an xml:lang attribute on a `<ti:translation>` element. And several of your translations don't have that. Change that and see if it passes. But you are certainly right that that is not a very helpful error message.